credential-free/spring-cloud-azure-starter-jdbc-mysql/spring-cloud-azure-mysql-sample/src/main/java/com/azure/spring/sample/mysql/TodoController.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.sample.mysql;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/")
+public class TodoController {
+
+    private final TodoRepository todoRepository;
+
+    public TodoController(TodoRepository todoRepository) {
+        this.todoRepository = todoRepository;
+    }
+
+    @PostMapping("/created")
+    public Todo createTodo() {
```

Comment:
I think we should use RequestParam or RequestBody to get the params.

Don't forget to update docs after modification.